### PR TITLE
aura api cloning

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by fiquick** [comment]
[Allow overwriting using API for CMK instances if destination is encrypted with the same CMK](https://trello.com/c/8IBHH9oS/357-allow-overwriting-using-api-for-cmk-instances-if-destination-is-encrypted-with-the-same-cmk)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 1]
```diff
@@ -55,10 +55,22 @@ For more information see the xref:auradb/importing/import-database.adoc#_neo4j_a
 === Clone an instance protected by CMK
 
 To clone an instance protected by a Customer Managed Key, the key must be valid and available to Aura.
-The cloned instance, by default, uses the available Customer Managed Key for that region and product.
+If the same CMK does not exist in the destination region and product, the cloned instance must be encrypted with an available CMK for that region and product, which you can choose.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:58]: What is it that you can choose? the key, the region, the product?

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 2]
```diff
@@ -55,10 +55,22 @@ For more information see the xref:auradb/importing/import-database.adoc#_neo4j_a
...
 
-It is best practice to use the same CMK key as the instance it’s being cloned from. 
-You can override this to use another CMK key--but you can not use the Neo4j Managed Key.
+It is best practice to use the same Customer Managed Key as the instance it’s being cloned from. 
+You can override this to use another Customer Managed Key - but you can not use the Neo4j Managed Key.
+
+When cloning an instance that is encrypted with a Customer Managed Key, specific restrictions apply when using the API.
+Below are the details and error messages that you may encounter depending on the cloning method and key configurations.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:64]:
```suggestion
Below are the details and possible errors that you may encounter depending on the cloning method and key configurations.
```

File: modules/ROOT/pages/platform/security/encryption.adoc [hunk 3]
```diff
@@ -55,10 +55,22 @@ For more information see the xref:auradb/importing/import-database.adoc#_neo4j_a
...
+
+.Summary of cloning restrictions
+|===
+| Cloning method   | Destination key            | Result
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/encryption.adoc:68]: The column _Destination key_ is misleading. I see what you are trying to say, but either change the title of the column or how you phrase the contents.

**Review by AlexicaWright — COMMENTED** [review]
Some clarifications needed, but good and useful content!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-475.surge.sh

Updated pages:
- [Encryption](https://neo4j-docs-aura-475.surge.sh/aura/platform/security/encryption)

**Pull request merged**